bluebrain/repo-bluebrain/packages/py-neurodamus/package.py

```diff
@@ -65,8 +64,21 @@ class PyNeurodamus(PythonPackage):
     depends_on("py-scipy", type=("build", "run"))
     depends_on("py-psutil", type=("build", "run"), when="@2.12.1:")
 
+    depends_on("py-setuptools", type=("build", "run"), when="@:3.1")
+    depends_on("py-hatchling", type="build", when="@3.2:")
+    depends_on("py-hatch-vcs", type="build", when="@3.2:")
+
+    @property
+    def datadir(self):
+        """Location of hoc and mod files"""
+        if self.spec.satisfies("@:3.1.1"):
+            return self.prefix.lib
+        return self.prefix.join(python_purelib).neurodamus.data
+
     @run_after("install")
     def install_files(self):
+        if self.spec.satisfies("@3.2:"):
+            return
```

Comment:
Why 3.2? We have to tag that version in the next release?

Reply:
I picked the next minor release, since we can only use >=, not > in version comparisons.

I can change it to 3.1.2 if you prefer.

Reply:
I just don't know if we will bump the minor or patch version in the next release so I prefer 3.1.2 here to have more options. What do you think ?

Reply:
OK, I'll change it!